west.yml

```diff
@@ -19,3 +21,9 @@ manifest:
       path: modules/lib/memfault-firmware-sdk
       remote: memfault
       revision: 1.8.1-esp32-zephyr
+    
+    - name: mcuboot
```

Comment:
Honestly pretty cool/hardcore to use the upstream version of MCUboot, I might instead just name-allowlist this above from zephyr though: 

https://github.com/zephyrproject-rtos/zephyr/blob/main/west.yml#L284

Reply:
Yeah I would recommend we use the Zephyr one, that's what upstream Zephyr tests with